flipkart_scraper.py

```diff
@@ -0,0 +1,120 @@
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+def get_url(dropdownselection,limit):
+  urlss=[] 
+  if dropdownselection==1:
+    for i in range(1,limit):  #use n+1 pages for clearing index out of range error
+      laps='https://www.flipkart.com/laptops/pr?sid=6bo,b5g&otracker=categorytree&page='+str(i)
+      urlss=urlss+[laps]
+  elif dropdownselection==2:
+    for i in range(1,limit): 
+      mobs='https://www.flipkart.com/mobiles/pr?sid=tyy%2C4io&otracker=categorytree&page='+str(i)
+      urlss=urlss+[mobs]
+  elif dropdownselection==3:
+     for i in range(1,limit): 
+      cams='https://www.flipkart.com/cameras/pr?sid=jek,p31&otracker=categorytree&page='+str(i)
+      urlss=urlss+[cams]
+  elif dropdownselection==4:
+     for i in range(1,limit): 
+      hpo='https://www.flipkart.com/headphones/pr?sid=fcn&otracker=categorytree&page='+str(i)
+      urlss=urlss+[hpo]
+  elif dropdownselection==5:
+     for i in range(1,limit): 
+      acs='https://www.flipkart.com/air-conditioners/pr?sid=c54&otracker=categorytree&page='+str(i)
+      urlss=urlss+[acs]
+      # Change the variable with a dropdown box when merging with its front end
+  return (urlss)
+
+
+def scrape_page(urlss,limit-1):
+  import requests     #Helps to send HTTP request(connections)
+  from urllib.request import urlopen as ureq #Helps to get connected with Specified URL's
+  from bs4 import BeautifulSoup as soup #Helps to extract webpages using HTML Parser function
+  import smtplib #Helps to send out mails using SMTP
+  import io  #Helps tp encode the characters into specified format
+  import csv  #Helps to read csv file
```

Comment:
A good practice is top have all of your import statements at the top of your file, instead of inside functions. You can also remove the comments because interested people can search for that on their own. (better looking)